governance.md

```diff
@@ -0,0 +1,164 @@
+Refer to the section on Community Groups and SIGs in particular for details on how this is performed.
+5. **Truly Open Community** : The best way to build a sustainable open source community is to ensure the community is a cohesive, inclusive, and equitable place where all the voices are heard and anyone can rise to leadership positions.
+
+### Code of Conduct
+
+We are using the [upstream OpenInfra Foundation Code of Conduct](https://openinfra.dev/legal/code-of-conduct) until we can convene the new Community SIG to lead an open effort to create our own, modern code of conduct.
+
+### Community membership
+**Note:** This document is a work in progress, especially these membership guidelines
+
+We are opening the Governance 1.0 directly borrowing from the [Kubernetes community group definitions](https://github.com/kubernetes/community/blob/master/community-membership.md), then reviewing and refining for the Governance 1.1.
+
+This doc outlines the various responsibilities of contributor roles in Op1st.
+The Op1st project is subdivided into subprojects under SIGs.
+Responsibilities for most roles are scoped to these subprojects.
+
+| Role | Responsibilities | Requirements | Defined by |
+| -----| ---------------- | ------------ | -------|
+| Member | Active contributor in the community | Sponsored by 2 reviewers and multiple contributions to the project | Op1st GitHub org member|
+| Reviewer | Review contributions from other members | History of review and authorship in a subproject | [OWNERS] file reviewer entry |
+| Approver | Contributions acceptance approval| Highly experienced active reviewer and contributor to a subproject | [OWNERS] file approver entry|
+| Subproject owner | Set direction and priorities for a subproject | Demonstrated responsibility and excellent technical judgement for the subproject | [sigs.yaml] subproject [OWNERS] file *owners* entry |
+
+### Community groups
+The Op1st community is initiating Governance 1.0 with only one type of community group, Special Interest Groups (SIGs).
+The purpose of doing so is to empower SIGs to gather interested people around a subject area, then decide and implement that into governance.
+For example, one of the first acts of the Community SIG is to establish a milestone schedule for defining and holding elections for future steering committee(s).
+Governance 1.1 will therefore include details on new community groups (subprojects and working groups).
+
+#### SIGs
+Special Interest Groups (SIGs) are persistent open groups that focus on a part of the project. SIGs must have open and transparent proceedings.
+Anyone is welcome to participate and contribute provided they follow the Op1st Code of Conduct.
+The purpose of a SIG is to own and develop a set of subprojects.
+
+A SIG can have its own policy for contribution, described in a README or CONTRIBUTING file in the SIG folder in this repo (e.g. sig-community/CONTRIBUTING.md), and its own mailing list, chat channel, etc.
+
```

Comment:
> MISSING SUB-SECTION - I forgot to include this, it should read like this:

#### Initiating SIGs

In order to get things started in a way to encourage transparency and participation, we are going to only initiate two SIGs to start, the Community SIG (`sig-community`) and the Operations SIG (`sig-ops`).
These two SIGs are immediately empowered through this governance to create new SIGs and/or subprojects.

To create subprojects, a SIG only needs to have consensus within itself.

To create a new SIG:

1. A SIG recognizes there is a need for a new SIG.
2. For example, the Community SIG decides to create the Data Science SIG right away, to formalize the already existing activity there.
3. Once the individual SIG is in consensus, the proposal is brought to all SIGs.
4. With all SIGs in consensus, the new SIG is formally created.

A similar process is followed when the Community SIG is ready to present recommendations for Governance 1.1 that creates committees.

Reply:
ACK. Add as a commit to the PR so we can comment 😄

Reply:
So that's how SIG are created, but what about SIG being deprecated ? EG, how would a SIG decide it is no longer important, if there is no longer anyone in it, etc, etc.

Reply:
> ACK. Add as a commit to the PR so we can comment smile

Done in d1adfd2 .